src/pages/service.tsx

```diff
@@ -180,7 +177,7 @@ const ServicePage: React.FC = () => {
                 messengerIcon,
                 "messengerDescription",
                 "messengerCTA",
-                "https://www.messenger.com/",
+                isMobile ? "fb://messaging/new" : "https://facebook.com",
```

Comment:
Maybe link to https://messenger.com/ on desktop?

Reply:
Originally it was that way. But the instructions are for the facebook client.